diff --git a/src/vs/code/browser/workbench/workbench.ts b/src/vs/code/browser/workbench/workbench.ts
index 534fe23..aed074b 100644
--- a/src/vs/code/browser/workbench/workbench.ts
+++ b/src/vs/code/browser/workbench/workbench.ts
@@ -402,6 +402,8 @@ class WindowIndicator implements IWindowIndicator {
 		else {
 			this.label = localize('playgroundLabel', "$(remote) Visual Studio Code Playground");
 			this.tooltip = localize('playgroundTooltip', "Visual Studio Code Playground");
+			this.label = localize('', "$(remote) OI Wiki Editor");
+			this.tooltip = "OI Wiki Editor";
 		}
 	}
 }
@@ -490,6 +492,7 @@ class WindowIndicator implements IWindowIndicator {
 			label: localize('learnMore', "Learn More")
 		}]
 	};
+	welcomeBanner;
 
 	// Window indicator (unless connected to a remote)
 	let windowIndicator: WindowIndicator | undefined = undefined;
@@ -527,7 +530,7 @@ class WindowIndicator implements IWindowIndicator {
 		settingsSyncOptions,
 		homeIndicator,
 		windowIndicator,
-		welcomeBanner,
+		// welcomeBanner,
 		productQualityChangeHandler,
 		workspaceProvider,
 		urlCallbackProvider: new PollingURLCallbackProvider(),
