diff --git a/src/vs/platform/product/common/product.ts b/src/vs/platform/product/common/product.ts
index 37d19f0..9a4d1eb 100644
--- a/src/vs/platform/product/common/product.ts
+++ b/src/vs/platform/product/common/product.ts
@@ -58,8 +58,8 @@ else {
 	if (Object.keys(product).length === 0) {
 		Object.assign(product, {
 			version: '1.63.0-dev',
-			nameShort: 'Code - OSS Dev',
-			nameLong: 'Code - OSS Dev',
+			nameShort: 'OI Wiki Editor',
+			nameLong: 'OI Wiki Editor',
 			applicationName: 'code-oss',
 			dataFolderName: '.vscode-oss',
 			urlProtocol: 'code-oss',
@@ -72,6 +72,11 @@ else {
 				'GitHub.remotehub',
 				'GitHub.remotehub-insiders'
 			],
+			extensionsGallery: {
+				serviceUrl: "https://marketplace.visualstudio.com/_apis/public/gallery",
+				cacheUrl: "https://vscode.blob.core.windows.net/gallery/index",
+				itemUrl: "https://marketplace.visualstudio.com/items"
+			},
 		});
 	}
 }
